appserver/extras/arquillian-containers/payara-micro/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
This file has no license

Reply:
This is a left-over from the other connector that shouldn't have been committed for this new one. Removed it.